src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml

```diff
@@ -43,27 +58,13 @@ sylius_fixtures:
                             dhl_express:
                                 code: "dhl_express"
                                 name: "DHL Express"
+                                channels:
+                                    - "US Web Store"
```

Comment:
Most of resources are gathered with code instead of name. We should change it to keep consistency